chalice/deploy/models.pyi

```diff
@@ -0,0 +1,124 @@
+from typing import List, Dict, Any, TypeVar, Union
```

Comment:
This file looks really similar to the `models.py`, it would be nice if this was easier to maintain.